### PR TITLE
Sort Typescript Keys

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Adds a new eslint rule to sort typescript keys

### CODE REVIEW

**Review by evansiroky — APPROVED** [review]
Looking good!

**Comment by evansiroky** [comment]
I'm assigning back to you to coordinate a final review and to merge once #3 is merged.

File: src/index.ts [hunk 1]
```diff
@@ -571,7 +571,7 @@ prog
       _: string[]
     }) => {
       if (opts._.length === 0 && !opts['write-file']) {
-        const defaultInputs = ['src', 'test'].filter(fs.existsSync)
+        const defaultInputs = [].filter(fs.existsSync)
```
Comment (binh-dam-ibigroup) [inline src/index.ts]: Wasn't this addressed? It looks some changes didn't make it to this PR.

**Comment by binh-dam-ibigroup** [comment]
@miles-grant-ibigroup Could you merge #3 into this branch please, so that changes from #3 aren't lost with this PR?

**Review by binh-dam-ibigroup — APPROVED** [review]

**Pull request merged**